app/src/main/java/ar/com/wolox/android/training/utils/CredentialsSession.java

```diff
@@ -0,0 +1,52 @@
+package ar.com.wolox.android.training.utils;
+
+import javax.inject.Inject;
+
+import ar.com.wolox.wolmo.core.di.scopes.ApplicationScope;
+import ar.com.wolox.wolmo.core.util.SharedPreferencesManager;
+
+/**
+ * CredntialsSession stores userData on sharedPreferences
+ */
+@ApplicationScope
+public class CredentialsSession {
+
+    private final String userKey = "USERNAME";
```

Comment:
This should be a constant (`private final static String ...`) and named like a constant (`USER_KEY`).